Makefile.system

```diff
@@ -873,6 +873,16 @@ endif
 endif
 endif
 
+ifeq ($(ARCH), csky)
+NO_BINARY_MODE	= 1
+BINARY_DEFINED	= 1
+endif
+
+ifeq ($(ARCH), ck860v)
```

Comment:
the ifeq (ARCH,ck860v) part looks a bit redundant here, assuming csky is going to be the ARCH and ck860v the TARGET (or eventually, one of several cpu targets under the csky architecture) ?